network-api/app/networkapi/templates/pages/landingpage.html

```diff
@@ -69,7 +69,7 @@ <h1 class="h1-white">{{ page.landingpage.header }}</h1>
             <div class="row">{% if page.landingpage.signup %}
               <div class="cms col-md-6">{{ page.landingpage.content | safe }}</div>
               <div class="col-md-5 offset-md-1">
-                <div class="join-us half-width col" data-cta-header="{{ page.landingpage.signup.header | escape }}" data-cta-description="{{ page.landingpage.signup.description | escape }}"></div>
+                <div class="join-us half-width col" data-cta-header="{{ page.landingpage.signup.header | escape }}" data-cta-description="{{ page.landingpage.signup.description | escape }}" data-newsletter="{{ page.landingpage.signup.newsletter }}"></div>
```

Comment:
don't know if you want to `escape` this as well?

Reply:
Nope, just a string here, not HTML.